docs/en/identifiers.rst

```diff
@@ -38,9 +38,9 @@ Configuration options:
 
 -  **fields**: The fields for the lookup. Default is
    ``['username' => 'username', 'password' => 'password']``. You can
-   also set the ``username`` to an array. For e.g. using
-   ``['username' => ['username', 'email'], 'password' => 'password']``
-   will allow you to match value of either username or email columns.
+   also set the lookup field name to an array. For e.g. using
+   ``['login' => ['username', 'email'], 'password' => 'password']``
```

Comment:
You can't change the key. The code checks for `username` https://github.com/cakephp/authentication/blob/3.x/src/Identifier/PasswordIdentifier.php#L96